src/functions/Invoke-AzOpsPush.ps1

```diff
@@ -177,8 +183,47 @@
             Write-PSFMessage @common -String 'Invoke-AzOpsPush.Change.AddModify.File' -StringValues $item
         }
         Write-PSFMessage @common -String 'Invoke-AzOpsPush.Change.Delete'
-        foreach ($item in $deleteSet) {
-            Write-PSFMessage @common -String 'Invoke-AzOpsPush.Change.Delete.File' -StringValues $item
+        if($DeleteSetContents -and $deleteSet) {
+            $allManagementGroups = (Get-AzManagementGroup).DisplayName
+            if(-not $allManagementGroups){
+                $deleteSetInclusionList = $deleteSet
+            }
+            foreach ($item in $deleteSet) {
+                Write-PSFMessage @common -String 'Invoke-AzOpsPush.Change.Delete.File' -StringValues $item
+                $inclusionFlag = $false
+                foreach($managementGroupName in $allManagementGroups){
+                    $path = Split-Path -Path $item
+                    $managementGroupName = $managementGroupName.ToLower()
+                    if($autoGeneratedPathFileName -ne "."){
+                        $path = $path.Replace($autoGeneratedPathFileName,"")
+                    }
+                    $path = (Split-Path -Path $Path -Leaf).ToLower()
+                    
+                    if($path -like "$managementGroupName*"){
```

Comment:
We need to support management groups

Reply:
Agree, will add this as a release.

Reply:
@daltondhcp - This has been incorporated in the PR now.